### DISCUSSION TITLE
Allow addresses to vote a portion of the token balance - not just 100% of a address' balance

### DISCUSSION THREAD

**Discussion by metrox-eth** [snapshot-labs/snapshot#750] [Ideas]
_Initially started by Scott K on Canny
September 27, 2021_

Certain wallet addresses represent many token holders e.g. exchanges. Those platforms cannot accurately represent the voting preferences of their users because addresses can only cast votes with 100% of the token balance. In order for these platforms to participate in governance on behalf of their users, they will need support for voting as a percentage of total balance and also to allow for voting a percentage across different options e.g. vote 20% of balance to "yes" and 30% of balance to "no"

**Comment by samwhosung** [comment]
This seems pretty important. Getting exchanges to use snapshot would be awesome!
What do you think @metrox-eth @bonustrack

**Comment by bonustrack** [comment]
This is not something we want to do, would be terrible if CEX vote for their users.

**Reply by samwhosung** [reply]
Why? This has nothing to do with exchanges voting for their users or not. They can do that anyway if they wanted. This is about exchange and other large wallet users being able to have a voice on snapshot.
